File: let's use some functions :D/K.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
    double r,x,y;
    cin >> r >> x >> y;
    double ans=sqrt((x*x)+(y*y));
    if (ans < r) cout << 2;
    else cout << ceil(ans/r);
}

File: let's use some functions :D/K.py
import math

R, X, Y = map(int, input().split())

D = math.sqrt(X**2 + Y**2)

if D % R == 0:
    steps = int(D // R)
elif D < R:
    steps = 2
else:
    steps = math.ceil(D / R)

print(steps)
